src/data.js

```diff
@@ -1,4 +1,4 @@
-const activities = [
+const data = [
```

Comment:
For `export default` the variable name doesn't actually have to match the name of what you're importing.

So you could have:
```js
// data.js
const foobar = [1,2,3]
export default foobar
```

Then in another file:
```js
import someRandomName from '../data.js'
// ... code
```

This only works when using `export default`.